includes/functions.php

```diff
@@ -1994,19 +1994,16 @@ function tsml_scan_data_source($data_source_url)
 					$term = get_term_by('term_id', $data_source_parent_region_id, 'tsml_region');
 					$parent_region = $term->name;
 					$message .= "parent_region: $parent_region <br>";
-					//$message .= "change_detect: $data_source_change_detect <br>";
-					$message .= " database count: $data_source_count_meetings <br>";
+					$message .= "database record count: $data_source_count_meetings <br>";
 					$feedCount = count($meetings);
-					$message .= "data source feed cnt: $feedCount<br>";
+					$message .= "data source feed count: $feedCount<br>";
 					$message .= "Last Refresh: <span style='color:red;'>*</span>" . Date("l F j, Y  h:i a", $data_source_last_import) . '<br>';
 					$message .= "<br><b><u>Detected Difference</b></u><br>";
 					$message .= "<table border='1' style='width:600px;'><tbody><tr><th>Update Mode</th><th>Meeting Name</th><th>Day of Week</th><th>Last Updated</th></tr>";
-					foreach	($meetings_updated as $key => $value) {
-						$message .= $value;
-					}
+					$message .= implode('', $meetings_updated);	
 					$message .= "</tbody></table><br>";
-					$import_page_url = 'https://' . $_SERVER['SERVER_NAME'] . '/wp-admin/edit.php?post_type=tsml_meeting&page=import';
-					$message .= "<a href='" . $import_page_url . "' style=' margin: 0 auto;background-color: #4CAF50;border: none;color: white;padding: 25px 32px;text-align: center;text-decoration: none;display: table;font-size: 18px;'>Goto Import & Settings page...</a>";
+					$import_page_url = 'https://' . admin_url() . '/wp-admin/edit.php?post_type=tsml_meeting&page=import';
```

Comment:
`admin_url()` doesn't work this way. you'll get `https://https://mywebsite.org/wp-admin/wp-admin/edit.php…`

try `$import_page_url = admin_url('edit.php?post_type=tsml_meeting&page=import');` instead

Reply:
Uhg... I didn't even test it out after making the change. Something came up and I didn't get back to it.  Thanks for catching...

Reply:
Tested with parameter in the call. Good to go.